---
# Feel free to add content and custom Front Matter to this file.
# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults

layout: default
---

## Plant Gene Discovery Platform

<p align="justify">
    The Plant Gene Discovery Platform is an open-source initiative focused on integrating genomic, transcriptomic, and regulatory data to accelerate the identification of biotechnologically relevant genes in non-model agricultural species. Designed for biotechnologists and bioinformaticians, the platform combines network analyses (such as gene co-expression networks and regulatory networks), providing a unified view of complex biological mechanisms, such as stress response. Its modular design allows continuous incorporation of public data (e.g., genomic annotations, SRA datasets) and computational tools, while APIs ensure interoperability between services, such as pipeline execution and database queries. With a focus on accessibility and reproducibility, the platform aims to overcome existing resource fragmentation, democratizing access to integrative analyses for agronomically significant species, such as soybean and eucalyptus. 
</p>

## ATLAS Project

<p align="justify">
    As the first outcome of the platform, ATLAS is a reproducible and scalable RNA-Seq analysis workflow for plants, automating processes from raw data acquisition (via portals like JGI and SRA) to transcript quantification using Salmon with decoy sequences to filter spurious alignments and enhance precision. Built on Nextflow and asynchronously managed by Celery with Redis, the pipeline integrates critical steps such as adaptive quality control (Trimmomatic), transcriptome indexing, and consolidated reporting (MultiQC), all within Docker-containerized environments. Its decoupled architecture enables API-driven execution, connecting directly to the platform’s database to store results like expression profiles, essential for constructing co-expression networks. Focused on agricultural species, ATLAS forms the foundation for downstream analyses on the platform, combining technical reproducibility with scalability for large-scale transcriptomic studies. Currently in its early implementation phase, the project represents a milestone in standardizing RNA-Seq analyses for plant research.
</p>

## TFFBR Classifier

<p align="justify">
A Deep Learning-Based Classification Approach for Identifying Transcription Factor Families Binding Regions in Plant Genomes.
</p>

### Publications

<p align="justify">
FLORES, Lorrana Verdi et al.. TITLE. In: X-Meeting presentations. Anais...João Pessoa(PB) Local, 2025. Available in: <a href="" target="_blank">xmeeting-2025-number</a>. Access in: 00/00/2025.
</p>

<p align="justify">
FLORES, Lorrana Verdi et al.. TITLE. In: X-Meeting presentations. Anais...João Pessoa(PB) Local, 2025. Available in: <a href="" target="_blank">xmeeting-2025-number</a>. Access in: 00/00/2025.
</p>
